mobile_back_tflite/cpp/backend_tflite/BUILD

```diff
@@ -29,6 +29,7 @@ cc_library(
     srcs = ["tflite_c.cc"],
     hdrs = [
         "tflite_settings.h",
+        "tflite_settings_apple.h",
```

Comment:
Should this be conditional on ios build?

Reply:
We could make this conditional, but in my opinion this would make build config more complicated without any benefits.

Reply:
@mcharleb are you OK with current solution or should I change it?

Reply:
I'm OK with it